backend/src/routes/actions.ts

```diff
@@ -16,4 +21,26 @@ router.post('/', (req: Request, res: Response) => {
   debug(req.body);
 });
 
+router.post('/playerSit', (req: Request, res: Response<PlayerSitResult>) => {
+  const body = req.body as PlayerSitPayload;
+  let seatNumber = body.selectedSeatNumber;
+  let clientId = body.socketId
+  // Try to sit at table
+  let join_room = PokerTable.sitAtTable('table_1', seatNumber, clientId);
```

Comment:
change these consts because we aren't redefining them and the error message should be here to send to back to client